docs/updating/update-to-43.md

```diff
@@ -16,8 +16,111 @@ modified_at: 2024-07-31
 
 ## Update to CKEditor&nbsp;5 v43.0.0
 
-_Released on August xx, 2024._
+_Released on August 7, 2024._
 
 For the entire list of changes introduced in version 43.0.0, see the [release notes for CKEditor&nbsp;5 v43.0.0](https://github.com/ckeditor/ckeditor5/releases/tag/v43.0.0).
 
-Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
\ No newline at end of file
+Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
+
+### Export to Word v2 becomes the default
+
+In version v42.0.0, we introduced new version of Export to Word plugin that utulizes updated v2 of the converter. It was introduced in an opt-in manner. From the v43.0.0, the Export to Word v2 is **the default** configuration. This may require updates to the editor's configuration if you use this plugin.
```

Comment:
```suggestion
In version v42.0.0, we introduced a new version of the Export to Word plugin that utilizes the updated v2 of the converter. It was introduced in an opt-in manner. From v43.0.0, the Export to Word v2 is **the default** configuration. This may require updates to the editor's configuration if you use this plugin.
```